generated/nidmm/session.py

```diff
@@ -1894,11 +1894,11 @@ def fetch_waveform(self, array_size, maximum_time=datetime.timedelta(millisecond
         '''
         vi_ctype = _visatype.ViSession(self._vi)  # case S110
         maximum_time_ctype = _converters.convert_timedelta_to_milliseconds(maximum_time, _visatype.ViInt32)  # case S140
-        array_size_ctype = _visatype.ViInt32(array_size)  # case S190
+        array_size_ctype = _visatype.ViInt32(array_size)  # case S210
         waveform_array_size = array_size  # case B600
         waveform_array_array = array.array("d", [0] * waveform_array_size)  # case B600
         waveform_array_ctype = get_ctypes_pointer_for_buffer(value=waveform_array_array, library_type=_visatype.ViReal64)  # case B600
-        actual_number_of_points_ctype = _visatype.ViInt32()  # case S200
+        actual_number_of_points_ctype = _visatype.ViInt32()  # case S220
```

Comment:
was this renumbering necessary?
If it was, it'd be great to get that out of the way in a separate PR that does just that in preparation for this.